tox.ini

```diff
@@ -67,6 +67,7 @@ max-line-length = 88
 extend-ignore = E203, E501
 per-file-ignores =
   __init__.py: F401
+ gallery/tutorials/configuration.py: T201, E402
```

Comment:
That is for the `import aspire`. You are right we don't need it, but if anyone ever added code above that line it would trip. I'm inclined to leave it in.

Reply:
oh wow good point yeah!